packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/features/exposurepoint/ExposurePointFeature.java

```diff
@@ -78,9 +80,18 @@ public void updateBuilder(@NonNull CaptureRequest.Builder requestBuilder) {
     if (!checkIsSupported()) {
       return;
     }
-    requestBuilder.set(
-        CaptureRequest.CONTROL_AE_REGIONS,
-        exposureRectangle == null ? null : new MeteringRectangle[] {exposureRectangle});
+
+    if (exposureRectangle != null) {
+      requestBuilder.set(
+          CaptureRequest.CONTROL_AE_REGIONS, new MeteringRectangle[] {exposureRectangle});
+    } else {
+      try {
+        requestBuilder.set(CaptureRequest.CONTROL_AE_REGIONS, null);
+      } catch (Exception e) {
+        throw new AssertionError(
```

Comment:
Maybe throw an `IllegalArgumentException` that indicates that an invalid value was passed for the device (since the crash on null seems device specific)?

Reply:
I can do this:
  catch (Exception e) {
         throw new IllegalArgumentException(
             "Failed to clear exposure rectangle for the ExposurePointFeature.", e);
       }
       
       I'm still getting used to the code :)

Reply:
No worries :) That sounds good to me!